students/ThanhNhan/lesson06/test_mailroom.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+import mailroom4 as mr4
+import os
+
+"""
+Send a thank you (adds a new donor or updates existing donor info)
+Create a report
+Send letters (creates files)
+Your unit tests should test the data manipulation logic code: generating thank you text, adding or updating donors, and listing donors.
+Send a thank you (adds a new donor or updates existing donor info)
+"""
+def test_existing_donor():
+
+    new_donor = {"Tom Cruise": [233.23, 12.17],
```

Comment:
great to have a dataset for the tests.

But better to have it outside this one test, so you can use it in other tests as well.